XRTK-Core/Packages/com.xrtk.core/Runtime/Definitions/MixedRealityRegisteredServiceProvidersProfile.cs

```diff
@@ -1,14 +0,0 @@
-﻿// Copyright (c) XRTK. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
-
-using UnityEngine;
-using XRTK.Definitions.Utilities;
-using XRTK.Interfaces;
-
-namespace XRTK.Definitions
-{
-    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Registered Service Providers Profile", fileName = "MixedRealityRegisteredServiceProvidersProfile", order = (int)CreateProfileMenuItemIndices.RegisteredServiceProviders)]
-    public class MixedRealityRegisteredServiceProvidersProfile : BaseMixedRealityServiceProfile<IMixedRealityExtensionService>
```

Comment:
I think we should keep this class but change the interface to `IMixedRealityService`

Reply:
Can you clarify? We can register services without this profile in the root profile? I thought it's the whole point.

Reply:
I see your point about having a "default" inspector @StephenHodgson , however, with nothing configured nothing will draw anyway.  So what is the point (other than it will draw the XRTK logo.)

Reply:
The root configuration profile only shows registered systems, which implies a single instance. We don't have a dedicated place to register ad hoc services.

Reply:
I'll update the Path in the CreateAssetMenu to be more specific 
But ultimately:

* Custom Systems - Are registerable through the editor along with their profiles
* Custom Services - Are not, and are only accessible through code.

Both have their uses.

Reply:
Restored and updated